README.md

```diff
@@ -1,17 +1,36 @@
-## My Project
+# AWS Distribution of Kubeflow
 
-TODO: Fill this README out!
+## Overview
 
-Be sure to:
+This repository contains configuration files and [overlays](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays) to deploy Kubeflow on [Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/).
 
-* Change the title in this README
-* Edit your repository description on GitHub
+## Getting Started
+
+There are a number of deployment options for installing Kubeflow with AWS service integrations. For detailed instructions regarding available AWS-integrated deployment options, see the [docs/deployment](docs/deployment) directory.
+
+## Help & Feedback
+
+For help, please consider the following venues (in order):
+
+* Documentation: Link TBD
```

Comment:
Shouldn't we just link to the current Kubeflow website for now instead of a TBD, just in case we ending up releasing this before docs are done.

Reply:
upstream docs are not in good shape right now. Hence I am refraining from linking atm. I would be more comfortable once we have a PR